client/galaxy/scripts/mvc/tag.js

```diff
@@ -133,7 +135,70 @@ var TagsEditor = Backbone.View
 });
 
 // =============================================================================
+
+var WorkflowTagsEditor = TagsEditor.extend({
```

Comment:
Yeah, I guess we could change the behavior of the tag editor (it's used for datasets and collections as well) to be the same as the WorkflowTagsEditor (i.e click on the tag to edit it, and close the editor with Escape instead of a separate button).

In the